### PR TITLE
Add icon to ert

### PR DESCRIPTION
**Pull request by oyvindeide**
As ert does not have an icon, this adds one.

Task manager:
image

Documentation:

image

README:

image

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# ert
+<div style="text-align: center;">
+<img width="400" src="src/ert/gui/resources/gui/img/ert_icon.svg">
```
Comment (sondreso) [inline README.md:2]: This is way too big

File: docs/index.rst [hunk 2]
```diff
@@ -1,3 +1,6 @@
+.. image:: ../src/ert/gui/resources/gui/img/ert_icon.svg
```
Comment (sondreso) [inline docs/index.rst:1]: Should either scale down or I believe the theme has builtin support for icon which should be used instead.

**Review by sondreso — CHANGES_REQUESTED** [review]
Should also clean up the lack of color between some of the lines: 
image

Does not work very well with varying background color

File: README.md (hunk 1 above)
Comment (oyvindeide) [reply README.md]: Using the same scale as numpy:

image

File: docs/index.rst (hunk 2 above)
Comment (oyvindeide) [reply docs/index.rst:1]: The icon support is only for some supported icons I believe?

**Comment by oyvindeide** [comment]
This PR: README:

image

vs main:

image

Docs:

This PR:

image

vs main:

image

Taskbar:

image

**Review by sondreso — APPROVED** [review]

**Pull request merged**